docs/odoo/usage/debug-assets-mode.rst

```diff
@@ -0,0 +1,19 @@
+================================
+How to activate debug assets mode 
+================================
+
+1.  Add ``debug=assets`` parameter to your url, e.g.: ::
+
+     localhost:8069/web?debug=assets#
+     
+2. Or use UI
+
+
+10.0, 11.0
+==========
```

Comment:
You made  a new section, while you intended this be part of ``2. Or use UI``.  
Any way, we already have a section how to do it via UI
http://odoo-development.readthedocs.io/en/latest/odoo/usage/debug-mode.html?highlight=developer